js/impress.js

```diff
@@ -703,6 +703,12 @@
         //   consistency I should add [shift+tab] as opposite action...
         document.addEventListener("keyup", function ( event ) {
 
+            ///press H button to redirect to the overview page,if your overview changes,
+            //just change the words after #/ for what your overview page's id.
+            if(event.keyCode==72){
+                window.location.replace(window.location.pathname+"#/overview");
```

Comment:
We can't couple `impress.js` with the reference of an external URL. The "overview" from the impress.js example is application specific and not mandatory.